futures-select-macro/LICENSE-APACHE

```diff
@@ -0,0 +1,202 @@
+                              Apache License
```

Comment:
The license files can be just symlinks to the ones in the root (as long as you don't care about publishing from windows working).